djangocms_text_ckeditor/templates/cms/plugins/widgets/ckeditor.html

```diff
@@ -50,9 +50,12 @@
 			});
 		}
 	}
+	function initCMSCKEditor() {
```

Comment:
This function must be created unique for each ckeditor instance in the page, please use ``{{ ckeditor_function }}`` template variable to do so